### PR TITLE
Marketplace frontend

### PR DESCRIPTION
**Pull request by christinang3030**

### CODE REVIEW

**Review by isabellapiz — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ function App() {
         <Routes>
           <Route path="/" element={<Home />} />
           <Route path ="/Profile" element={<Profile />} />
+          <Route path ="/Marketplace" element={<Marketplace />} />
```
Comment (JoshuaLawson8) [inline src/App.js]: will have to manually merge this

File: src/components/Marketplace/Marketplace.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {Navigation} from "../index";
+import PTProfile from "../Profile/PTProfile";
+
+function Marketplace() {
```
Comment (JoshuaLawson8) [inline src/components/Marketplace/Marketplace.jsx]: no functionality, maybe we should discuss what funct will look like in next meeting?

**Review by JoshuaLawson8 — APPROVED** [review]
good, branches will have to be somewhat manually merged because of many changes to Appjs

File: server/endpoints/models/Client.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  bio: {
+    type: String,
+  },
+  postiveRatingCount: {
+    type: Number
+  },
+  negativeRatingCount: {
+    type: Number
+  },
+  location: {
```
Comment (euniceyoh) [inline server/endpoints/models/Client.js]: did you want to use GeoJSON for location?

File: server/endpoints/models/PT.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    type: String,
+    required: true
+  },
+  postiveRatingCount: {
+    type: Integer
+  },
+  negativeRatingCount: {
+    type: Integer
+  },
+  location: {
```
Comment (euniceyoh) [inline server/endpoints/models/PT.js]: same question here for GeoJSON

**Review by euniceyoh — CHANGES_REQUESTED** [review]

File: server/endpoints/models/Client.js (hunk 3 above)
Comment (christinang3030) [reply server/endpoints/models/Client.js]: Yes, I think that would be great.

File: server/endpoints/models/PT.js (hunk 4 above)
Comment (christinang3030) [reply server/endpoints/models/PT.js]: Yes!

File: server/endpoints/models/Client.js (hunk 3 above)
Comment (charlotte-zhuang) [reply server/endpoints/models/Client.js]: This change is in `main` now.

File: server/endpoints/models/PT.js (hunk 4 above)
Comment (charlotte-zhuang) [reply server/endpoints/models/PT.js]: This change is in `main` now.

File: src/App.jsx [hunk 5]
```diff
@@ -18,6 +18,11 @@ import {
   NotFound,
   UserDashboard,
   Chat,
+  Profile,
+  Marketplace
+} from './components';
+import Request from './components/Requests/Request';
```
Comment (euniceyoh) [inline src/App.jsx]: i think request is alr imported in line 30

**Comment by euniceyoh** [comment]
i think some of the links in the nav bar disappear when you're on marketplace. i think profile is also blank, was that intentional?

but marketplace is looking really cool!

**Review by charlotte-zhuang — APPROVED** [review]
made some changes to fix console errors and search behavior

File: src/App.jsx (hunk 5 above)
Comment (charlotte-zhuang) [reply src/App.jsx]: fixed

**Review by euniceyoh — APPROVED** [review]
looks good

**Pull request merged**